src/auth/activation.ts

```diff
@@ -48,10 +48,10 @@ async function showManageConnectionsOnStartup() {
         return
     }
 
-    if (isInDevEnv()) {
-        getLogger().debug('firstStartup: Detected we are in Dev Env, skipping showing Add Connections page.')
+    if (isCloud9('any')) {
```

Comment:
i think it was intended that any kind of devenv (not only cloud9 devenvs) should not show this page. 

```suggestion
    if (isInDevEnv() || isCloud9('any')) {
```

Reply:
Thanks for catching, the vscode connected dev env slipped my mind